build/tasks/GeneratePackageVersionPropsFile.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Are this and GenerateRestoreSourcesPropsFile.cs needed anymore now that we use `dependencies.props`?

Reply:
Good catch. GeneratePackageVersionPropsFile is redundant. This can come from build tools instead of repo tasks. GenerateRestoreSourcesPropsFile.cs on the other hand is still unique to repotasks